runtime/go.mod

```diff
@@ -40,6 +41,7 @@ require (
 	github.com/cyphar/filepath-securejoin v0.2.3 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/emicklei/go-restful/v3 v3.8.0 // indirect
+	github.com/evanphx/json-patch v4.12.0+incompatible // indirect
```

Comment:
Can you please update `controller-check` to `evanphx/json-patch/v5` do we don't downgrade it for all of runtime to a pre Go mod version?

Reply:
That version of json-patch is no longer introduced by this change and exists in the main branch. It's a dependency of client-go. Marking this as resolved.